### DISCUSSION TITLE
Recursos de CSS ()

### DISCUSSION THREAD

**Discussion by OscarPedroza** [LaunchX-InnovaccionVirtual/FrontEnd-Mision#275] [Ideas]
Buenas noches compañeros, les comparto algunos enlaces con información sobre CSS para conocer como funciona un poco más en profundidad algunos temas como lo son selectores, la cascada y herencia, cómo calcular la especificidad de un elemento, el box model, posicionamiento de cajas, etc.

Formas de incrustar CSS en una página web y sus ventajas y deventajas: 
[https://revistadigital.inesem.es/informatica-y-tics/incluir-css-en-html/](https://revistadigital.inesem.es/informatica-y-tics/incluir-css-en-html/)

Selectores: 
[https://www.w3.org/wiki/CSS_/_Selectores_CSS](https://www.w3.org/wiki/CSS_/_Selectores_CSS)

Cascada y herencia: 
[https://developer.mozilla.org/es/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance](https://developer.mozilla.org/es/docs/Learn/CSS/Building_blocks/Cascade_and_inheritance)

Qué es y como calcular la especificidad: 
https://codigofacilito.com/articulos/especificidad-css
[https://webdesign.tutsplus.com/es/tutorials/what-is-css-specificity--cms-34141](https://webdesign.tutsplus.com/es/tutorials/what-is-css-specificity--cms-34141)

Box model: 
[https://developer.mozilla.org/es/docs/Web/CSS/CSS_Box_Model](https://developer.mozilla.org/es/docs/Web/CSS/CSS_Box_Model)

Position:
[https://uniwebsidad.com/libros/css/capitulo-5/posicionamiento](https://uniwebsidad.com/libros/css/capitulo-5/posicionamiento)

A su vez les comparto dos cursos de desarrollo web bastante completos y donde se explica con gran detalle, uno es de HTML y CSS respectivamente: 
[https://www.youtube.com/watch?v=vz4z0RLcAyk&ab_channel=DorianDesings](https://www.youtube.com/watch?v=vz4z0RLcAyk&ab_channel=DorianDesings)
[https://www.youtube.com/watch?v=N8V5JhasaSE&ab_channel=DorianDesings](https://www.youtube.com/watch?v=N8V5JhasaSE&ab_channel=DorianDesings)

Espero y les sirvan estas referencia. Saludos!
Si tienen sugerencias o más enlaces de apoyo me las pueden hacer saber, gracias.

**Comment by HeribertoBonilla** [comment]
Muchas gracias!
